src/browser-frontend/views/browser/chrome/tabbar/tab/tab-favicon.jsx

```diff
@@ -48,7 +50,14 @@ export default class TabFavicon extends PureComponent {
         mode="contain"
       />);
     } else if (this.props.loadState === DomainPageMetaModel.LOAD_STATES.LOADED) {
-      if (this.props.favicon !== undefined) {
+      if (this.props.url === Endpoints.BLANK_PAGE) {
+        contents = (<FittedImage
+          src="var(--theme-app-icon-32)"
```

Comment:
Should this logic be encoded into the getPageFavicons selector instead of the component?